### ISSUE TITLE
Project Proposal Peer Review - Group E

### ISSUE THREAD

**Issue by EmThanhLamHet** [khaukhau/COMP4010#2]
Overall, the project seems to have covered all requirements. It has also planned out all the steps taken to answer the two questions. However, I still have some comments about the project proposal:
- Not enough description in the first part. Data types is not presented, and there is inadequate description on each field of the data. The data field name is also not presented in the raw context.
- I suggest using multiple line graphs for trends in multiple locations, preferably locations with the greatest number of tornadoes for better visualization for the first question.
- For Mini Question 1 - Q2, I'd recommend using a Bar-Line combined plot for visualization of top 10 states with the most tornadoes, with the bar plot for the number of tornadoes and the line plot for fatalities, it would be a better choice than stacked bar chart.

**Comment by tienvu95** [comment]
I like your comments on the data description. While their proposal is very detailed, it's hard to follow as they refer to many variables in the dataset without explaining it in the document --> we have to refer back to the original doc. Moreover, I also agree with our points on mini question 1 q2, stacked bar chart doesn't make sense and line bar chart = better choice

**Comment by thaohp113** [comment]
Thank you all for your comments, they are all valuable. We have made the according adjustments:
- Data description: We have included the full description of each variable, such as name, data type and description, as well as some notes, for a comprehensive proposal. 
- Q2 – Mini question 1: We also agree that stacked bar chart may not be the optimal option, hence changing it to a line-bar chart.
